panc/src/main/java/org/quattor/pan/dml/data/ListResource.java

```diff
@@ -137,14 +143,21 @@ public Element put(int index, Element newValue) {
 
 		Element oldValue = null;
 
+        int size = list.size();
+
 		if (index < 0) {
-			throw new EvaluationException(MessageUtils.format(
-					MSG_INVALID_LIST_INDEX, Integer.toString(index)));
+            if (index < -size) {
```

Comment:
Indent more? Or are you trying to save line space?

Reply:
this is tabs vs 4space. i can do a cleanup of all whitespace in later PR, but not here. some other PR already depends on it.

Reply:
Ok.